Translations/Translations.csv

```diff
@@ -0,0 +1,88 @@
+"help_info_category","Type `{0}help [category]` for more info on a category.","Gebruik `{0}help [categorie]` voor meer info over een categorie."
+"help_info_command","Type `{0}help [command]` for more info on a command.","Gebruik `{0}help [commando]` voor meer info over een commando."
+"help_info_arguments","<Arguments> are required.\n[Arguments] are optional.","<parameters> zijn verplicht.\n[parameters] zijn optioneel."
+"help_brief","Print the help list.","Toon de help info."
+"help_help","Print the help list\n\n\u2003[command or category] \u2003 The command or category you need help on.","Toon de help info\n\n\u2003[commando of categorie] \u2003 Het commando of de categorie waarover je meer info wilt."
+"help_usage","\u2000[command or category]","\u2000[commando of categorie]"
+"membercount_usage","[channel name]","[kanaal naam]"
+"membercount_brief","performs a member count.","Telt het aantal leden."
+"membercount_help","Perform a member count.\n\nWithout parameter: Returns the server member count.\n\nWith a channel name: Returns a count of all members currently in the given channel.\n\nWith parameter 'online': Return all members currently online.","Telt het aantal leden.\n\nZonder parameter: Telt het aantal leden in de server.\n\nMet een kanaal: Telt het aantal leden in dat kanaal.\n\nMet parameter 'online': Telt het aantal leden die momenteel online zijn."
+"membercount_online_result","There are {0} people online in this server!","Er zijn momenteel {0} gebruikers online in deze server!"
+"membercount_server_result","There are {0} people in {1}!","Er zitten momenteel {0} gebruikers in {1}!"
+"membercount_channel_nonexistant","The provided channel does not exist, foemp!","Het kanaal dat je meegaf bestaat niet, foemp!"
+"membercount_empty_channel","<#{0}> is empty, foemp!","<#{0}> is leeg, foemp!"
+"membercount_single_person","There is 1 person in channel <#{0}>!","Er is 1 persoon aanwezig in <#{0}>!"
+"membercount_channel_result","There are {0} people in channel <#{1}>!","Er zijn {0} mensen aanwezig in <#{1}>"
+"list_err_does_not_exit","This list does not exist, foemp!","Deze lijst bestaat niet, foemp!"
+"list_err_already_subscribed","<@{0}>, you are already subscribed to {1}, foemp!","<@{0}>, je ontvangt al notificaties voor {1}, foemp!"
+"list_err_not_subscribed","<@{0}>, you are not subscribed to {1}, foemp!","<@{0}>, je ontvangt geen notificaties voor {1}, foemp!"
+"list_subscribed","Subscribed <@{0}> to {1}.","<@{0}> ontvangt nu notificaties voor {1}."
+"list_unsubscribed","Unsubscribed <@{0}> from {1}.","<@{0}> ontvangt niet langer notificaties voor {1}."
+"list_err_empty","Nobody to notify, foemp!","Deze lijst is leeg, foemp!"
+"notifying","**{0}** Notified by <@{1}>\n","Ahem, dit is NerdlandBot uit de montagekamer, willen de volgende personen zich op vraag van <@{1}> melden voor **{0}** a.u.b.\n"
```

Comment:
hmm difficult one.
The issue here is that in the english version this joke is not included, and both should have the same amount of parameters.
The joke also kind of ruins itself if you change it to another name....

Any suggestion to fix this?

Reply:
If you put `{2}` in the Dutch key instead of NerdlandBot, and then you call `translate("notifying", await culture(ctx)).format(list_name.capitalize(), ctx.message.author.id,ctx.guild.get_member(ctx.bot.user.id).display_name)` , the 3rd argument will simply be ignored in the English translation because it only uses arguments `{0}` and `{1}`. But the name will be printed in the Dutch translation because that one uses argument `{2}`.

Reply:
Maybe the joke would be a bit ruined by changing the name, but if you make it variable, it will change if the name of NerdlandBot is changed for some reason. And if you would use the code for a bot in a non-Nerdland-related server, the joke would be kind of weird to begin with, no matter what the name of the bot is. Maybe it is an idea to make a "Nerd-nl" translation separate from a Dutch one, that has mostly the same keys, but the Dutch one would not contain Nerdland specific jokes.

Reply:
could be an idea for the future indeed, for now, other servers will have to suffer the jokes

Reply:
the parameter was added though

Reply:
If I put this line `"notifying","**{0}** Notified by <@{1}>\n","Ahem, dit is {2} uit de montagekamer, willen de volgende personen zich op vraag van <@{1}> melden voor **{0}** a.u.b.\n"` in translations.csv, it works as expected on my computer. 

https://realpython.com/python-formatted-output/: " On the other hand, it’s fine if the arguments outnumber the replacement fields. The excess arguments simply aren’t used:"